tests/sessionid_filter/Makefile

```diff
@@ -0,0 +1,8 @@
+TARGETS=$(patsubst %.c,%,$(wildcard *.c))
+
+LDLIBS += -lpthread
+
+all: $(TARGETS)
+clean:
+	rm -f $(TARGETS)
+
```

Comment:
Let's rename this directory to "filter_sessionid" (see my comments on the exclude filter patch for the testsuite).  Also, don't forget to wire this into the top level Makefile so it gets executed.